### ISSUE TITLE
Contrast for "Select files to upload" is too low

### ISSUE THREAD

**Issue by dannycolin** [mozilla/send#1482]
The contrast ratio for the "Select files to upload" is on 2.69 and doesn't meet the minimum for WCAG AA.
